pkgs/applications/editors/rcptt/default.nix

```
@@ -0,0 +1,85 @@
+# Intende useage
+# Build the elexis-congo app via mvn -V clean verify -f elexis-congo/pom.xml
+# nix-build Elexis_Congo.nix
+# start one of the binaries under result/bin/
+#
+{ lib, pkgs, stdenv, bundlerEnv, ruby, makeWrapper, makeDesktopItem
+, gnome, git, gtk3, jdk11, fontconfig, freetype, glib
+, gsettings-desktop-schemas, webkitgtk, docutils, perl, xorg, zlib
+, libXrender, libsecret, unzip, rsync, fetchzip, ... }:
+let
+  desktopItem = makeDesktopItem {
+    name = "RCPTT";
+    exec = "rcptt";
+    icon = "rcptt";
+    comment = "RCP Testing Tool is a project for GUI testing automation of Eclipse-based applications. RCPTT is fully aware about Eclipse Platform's internals, hiding this complexity from end users and allowing QA engineers to create highly reliable UI tests at great pace.";
+    desktopName = "RCPTT";
+    genericName = "RCP Testing Tool";
+    categories = [ "Development" "IDE" "Java"  ];
+  };
+in
+
+stdenv.mkDerivation rec {
+  pname = "RCPTT";
+  version = "2.5.3" ;
+  name = "${pname}-${version}";
+  src = fetchzip {
+    url = "https://download.eclipse.org/rcptt/release/${version}/ide/rcptt.ide-${version}-linux.gtk.x86_64.zip";
```

Can we somehow parametrize this so that the package could be built for multiple systems?
```suggestion
    url = "https://download.eclipse.org/rcptt/release/${version}/ide/rcptt.ide-${version}-linux.gtk.${system}.zip";
```

---

Gotcha.